### DISCUSSION TITLE
멘토님의 MBTI는?

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [ThinkAboutSoftware/AcademicConference#14] [General]
[검사링크](https://www.16personalities.com/ko/%EB%AC%B4%EB%A3%8C-%EC%84%B1%EA%B2%A9-%EC%9C%A0%ED%98%95-%EA%B2%80%EC%82%AC)

여기에 댓글 남겨주세요!

**Comment by yoonucho** [comment]
intj

**Comment by hi-hyein** [comment]
intj

**Comment by Joseph-Cha** [comment]
INTJ !!

**Reply by Joseph-Cha** [reply]
https://www.16personalities.com/ko/성격유형-istj

**Comment by jongfeel** [comment]
![image](https://user-images.githubusercontent.com/17442457/115059232-382ad380-9f21-11eb-97de-3b0a61e3dc2b.png)

**Reply by Joseph-Cha** [reply]
https://www.16personalities.com/ko/성격유형-enfj

**Comment by hi-hyein** [comment]
https://www.16personalities.com/ko/%EC%84%B1%EA%B2%A9%EC%9C%A0%ED%98%95-infp

**Reply by yoonucho** [reply]
https://www.16personalities.com/ko/%EC%84%B1%EA%B2%A9%EC%9C%A0%ED%98%95-infj
